scripts/add-public-ssh-key.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
 user_public_key=$1
 if [ "$user_public_key" != "None" ] ; then
-    echo "$user_public_key" >> $HOME/.ssh/authorized_keys
+  if [ ! -f ${HOME}/.ssh/authorized_keys ]; then
+    mkdir ~/.ssh
```

Comment:
You should probably do `mkdir -p ~/.ssh` here, since non-existance of authorized_keys does not guarantee existence of ~/.ssh
without `-p` mkdir will fail if .ssh already exists